05-onchain.md

```diff
@@ -325,38 +325,44 @@ transaction output.
 
 There are no delays constraining node behavior in this case, so it's simpler for
 a node to handle than the case in which it discovers its local commitment
-transaction.
+transaction (see [Unilateral Close Handling: Local Commitment Transaction](#unilateral-close-handling-local-commitment-transaction)).
 
 ## Requirements
 
-When a node discovers a commitment transaction from the *remote node*:
-
-1. `to_remote`: No action is required; this is a simple P2WPKH output to us.
-   This output is considered *resolved* by the commitment transaction itself.
-2. `to_local`: No action required; this is a payment to them. This output is considered *resolved*
-   by the commitment transaction.
-3. HTLCs offered by the local node: See "HTLC Output Handling: Remote Commitment, Local Offers" below.
-4. HTLCs offered by the remote node: See "HTLC Output Handling: Remote Commitment, Remote Offers" below.
-
-A node MUST handle the broadcast of any *valid* commitment transaction
-from the *remote node* in this way; if it is unable to do so it MUST warn
-about lost funds.
+A local node:
+  - upon discovering a *valid* commitment transaction broadcast by a
+  *remote node*:
+    - if it is able to handle the broadcast:
```

Comment:
Perhaps we should remove the "if it is able" clause, and change:

    - MUST handle it as specified below

to:
    - if possible:
      - MUST handle each output as as specified below.
   - Otherwise:
     - MUST send warning regarding lost funds.
?

Reply:
sounds good